src/unix/fs.c

```diff
@@ -254,6 +254,37 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 }
 
 
+static ssize_t uv__fs_lutime(uv_fs_t* req) {
+#if defined(__linux__)                                                         \
+    || defined(_AIX71)                                                         \
+    || defined(__sun)                                                          \
+    || defined(__HAIKU__)
```

Comment:
Style: libuv house style is to put the `||` at the end of the line, not at the beginning.